<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-GAD15EJN" name="interfacetest.Jktest" tests="15" failures="0" timestamp="18 Dec 2019 06:20:06 GMT" time="0.262" errors="3">
  <testcase name="Tjaddress" time="0.017" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.013" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.014" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.013" classname="interfacetest.Jktest">
    <error type="java.lang.Exception" message="expectedValue doesn&#039;t match actualValue,actual Value is :success; expected Value is :请求参数不正确">
      <![CDATA[java.lang.Exception: expectedValue doesn't match actualValue,actual Value is :success; expected Value is :请求参数不正确
	at com.edu.utils.Checker.verify(Checker.java:64)
	at com.edu.utils.Checker.verifyXpath(Checker.java:47)
	at interfacetest.Jktest.Tjaddress(Jktest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- Tjaddress -->
  <testcase name="Tjaddress" time="0.014" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.014" classname="interfacetest.Jktest">
    <error type="java.lang.Exception" message="expectedValue doesn&#039;t match actualValue,actual Value is :success; expected Value is :receiverName参数类型不正确">
      <![CDATA[java.lang.Exception: expectedValue doesn't match actualValue,actual Value is :success; expected Value is :receiverName参数类型不正确
	at com.edu.utils.Checker.verify(Checker.java:64)
	at com.edu.utils.Checker.verifyXpath(Checker.java:47)
	at interfacetest.Jktest.Tjaddress(Jktest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- Tjaddress -->
  <testcase name="Tjaddress" time="0.019" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.020" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.014" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.017" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.024" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.025" classname="interfacetest.Jktest">
    <error type="java.lang.Exception" message="expectedValue doesn&#039;t match actualValue,actual Value is :success; expected Value is :receiverName参数类型不正确">
      <![CDATA[java.lang.Exception: expectedValue doesn't match actualValue,actual Value is :success; expected Value is :receiverName参数类型不正确
	at com.edu.utils.Checker.verify(Checker.java:64)
	at com.edu.utils.Checker.verifyXpath(Checker.java:47)
	at interfacetest.Jktest.Tjaddress(Jktest.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- Tjaddress -->
  <testcase name="Tjaddress" time="0.012" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.013" classname="interfacetest.Jktest"/>
  <testcase name="Tjaddress" time="0.033" classname="interfacetest.Jktest"/>
</testsuite> <!-- interfacetest.Jktest -->
